server/src/test/java/io/crate/integrationtests/SwapTableITest.java

```diff
@@ -63,6 +64,7 @@ public void testSwapTwoTablesWithDropSource() {
                                 4041));
     }
 
+    @Repeat(iterations = 50)
```

Comment:
Unrelated?

Reply:
Thanks for catching this. I will take this out..